warehouse/admin/templates/admin/users/account_recovery/initiate.html

```diff
@@ -0,0 +1,108 @@
+<li class="breadcrumb-item active">Initiate Account Recovery</li>
+{% endblock %}
+
+{% block content %}
+<form action="" method=POST>
+  <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
+  <div class="card">
+    <div class="card-header with-border">
+      <h3 class="card-title">1. Confirm user details</h3>
+    </div>
+    <div class="card-body">
+      <p class="card-text">
+        Confirm that TOTP or WebAuthn is enabled, that recovery codes have been generated.
+      </p>
+      <table class="table table-hover">
+        <tr>
+          <th>TOTP Enabled</th>
+          <th>{% if user.totp_secret %}<i class="fa fa-check text-green"></i>{% else %}<i class="fa fa-times text-red"></i>{% endif %}</th>
+        </tr>
+        <tr>
+          <th>WebAuthn Enabled</th>
+          <th>{% if user.webauthn %}<i class="fa fa-check text-green"></i>{% else %}<i class="fa fa-times text-red"></i>{% endif %}</th>
+        </tr>
+        <tr>
+          <th>Recovery Codes Generated</th>
+          <th>{% if user.has_recovery_codes %}<i class="fa fa-check text-green"></i>{% else %}<i class="fa fa-times text-red"></i>{% endif %}</th>
+        </tr>
+        <tr>
+          <th>Recovery Codes Used</th>
+          {% set n_codes = user.recovery_codes.all()|length %}
+          {% set n_remaining = user.recovery_codes.filter_by(burned=None).all()|length %}
+          <th>{{ n_codes - n_remaining }}/{{ n_codes }}</td>
+        </tr>
+      </table>
+    </div>
+  </div> <!-- .card -->
+
+  <div class="card">
+    <div class="card-header with-border">
+      <h3 class="card-title">2. Provide support issue link</h3>
+    </div>
+    <div class="card-body">
+      <div class="input-group input-group-lg">
+        <input name="support_issue_link" type="text" class="form-control input-lg" placeholder="https://github.com/pypi/support/issues/..."{% if query %} value="{{ query }}"{% endif %}>
+      </div>
+    </div>
+  </div>
+
+  <div class="card">
+    <div class="card-header with-border">
+      <h3 class="card-title">3. Select a GitHub repository</h3>
+    </div>
+    <div class="card-body">
+    {% if repo_urls %}
+    {% for project_name, repo_urls in repo_urls.items() %}
+    {% if repo_urls %}
+    <div class="form-check">
+      <input class="form-check-input" type="radio" name="project_name" value="{{ project_name }}" name="{{ project_name }}">
+      <label class="form-check-label" for="{{ project_name }}"><a href="{{ request.route_url("admin.project.detail", project_name=project_name) }}">{{ project_name }}</a></label>
+      <ul>
+      {% for kind, repo_url in repo_urls %}
+      <li>{{ kind }}: <a href="{{ repo_url }}" data-check-link-url="{{ request.camo_url(repo_url) }}"><i class="fa fa-question"></i> {{ repo_url }}</a></li>
+      {% endfor %}
+      </ul>
+    </div>
+    {% endif %}
+    {% endfor %}
+    {% else %}
```

Comment:
nit: add indentations for nesting visibility

I have a local branch for a better template linter/formatter, but that's a non-trivial lift to incorporate, so I haven't proposed that yet.

Reply:
89f7dc18f1d4f6ba3ebecd0e24b410ac2ce27583